integration-tests/sql/src/test/java/org/apache/camel/quarkus/component/sql/it/SqlTest.java

```diff
@@ -224,16 +228,20 @@ private ValidatableResponse postMapWithParam(String toUrl, String param, String
                 .then();
     }
 
-    private void route(String routeId, String operation, String expectedOutput) {
-        RestAssured.given()
-                .get("/sql/route/" + routeId + "/" + operation)
-                .then().statusCode(204);
-
-        if (expectedOutput != null) {
-            await().atMost(5, TimeUnit.SECONDS).until(() -> RestAssured
-                    .get("/sql/route/" + routeId + "/status")
-                    .then()
-                    .extract().asString(), equalTo(expectedOutput));
+    @SuppressWarnings("unchecked")
+    public static org.hamcrest.Matcher<java.util.Map<String, Object>> matchMapIgnoringCase(Map<String, Object> map) {
+        Matcher m = null;
+        for (Map.Entry<String, Object> entry : map.entrySet()) {
+            if (entry.getKey().toUpperCase().equals("PROCESSED")) {
+                //it is boolean type and different dbs return different representations of boolean
```

Comment:
Couldn't we have a loose assertion checking that it maches "0" or "false" ?

Reply:
I agree, I chose this way, because the assertions look complex. I'll try to cover boolean with it and we will see, whether its complexity is acceptable.

Reply:
Looks better then I thought! Done